engine/src/main/java/org/terasology/rendering/opengl/GraphicState.java

```diff
@@ -515,28 +470,6 @@ public void setRenderBufferMask(FBO fbo, boolean color, boolean normal, boolean
     }
 
     /**
-     * Sets the viewport of the currently bound FBO to the size of the Display.
-     * 
-     * Which might mean the full screen, the full size of the window or the
-     * currently set screenshot size if a screenshot is being taken.
-     */
-    // TODO: verify if this can become part of the bindDisplay() method.
-    public void setViewportToWholeDisplay() {
-        glViewport(0, 0, fullScale.width(), fullScale.height());
-    }
```

Comment:
Scrap that. I can imagine it's not straightforward to get the content of fullScale. We can leave with this for a while.